### PR TITLE
Version 0.0.2

### PR DESCRIPTION
**Pull request by ijunaid8989**

### CODE REVIEW

**Comment by ijunaid8989** [comment]
@hauleth any update on this, please?

**Comment by hauleth** [comment]
Could you not update the versions of the package? Leave that to the maintainer.

File: mix.exs [hunk 1]
```diff
@@ -4,7 +4,7 @@ defmodule ReviewdogCredo.Mixfile do
   def project do
     [
       app: :reviewdog_credo,
-      version: "0.0.1",
+      version: "0.0.2",
```
Comment (hauleth) [inline mix.exs]:
```suggestion
      version: "0.0.1",
```

**Comment by ijunaid8989** [comment]
@hauleth thanks,

now to use it? should I update anything?

**Pull request merged**